components/team_sidebar/components/team_button.jsx

```diff
@@ -138,7 +149,7 @@ class TeamButton extends React.Component {
             );
 
             // if this is not a "special" team button, give it a context menu
-            if (!this.props.url.endsWith('create_team') && !this.props.url.endsWith('select_team')) {
+            if (isNotSpecialButton) {
```

Comment:
We could just say 'isNotCreateTeamButton` to be more explicit. I don't think there's another type of button in the team button area.